GMnetENGINE.gmx/scripts/htme_roomend.gml

```diff
@@ -68,6 +69,21 @@ for(var i=0; i<ds_map_size(mapToUse); i+=1) {
                      instance_destroy();
                    }              
                }
+            } else {

```

Comment:
Why didn't I see this earlier??? We must clean non-persistent instances too. We keep them apart to handle them differently if needed in the future.